test/system/030-run.bats

```diff
@@ -1382,4 +1382,19 @@ search               | $IMAGE           |
     run_podman container rm $cname
 }
 
+# the /run should not be mounted as 777 in the init container
+@test "podman run - the /run is mounted as 777 in the init container with crun runtime" {
+    runtime=$(podman_runtime)
+    cname=$(random_string)
+    init_img="registry.access.redhat.com/ubi8/ubi-init"
+
+    run_podman run --runtime $runtime --name $cname -d --rm --network none --cap-add all $init_img /usr/sbin/init
```

Comment:
I "think" it should also work with `--systemd=always $IMAGE`.  This way we'd avoid pulling down an image.

@edsantiago WDYT?

Reply:
Yes, please try never to pull anything in tests. Other problems too, I'll send more feedback soon.

Reply:
Done, thank you for your review!